azerothcore/azerothcore-wotlk
# fix(DB/Creature): Fixed Fel'zerul RP on The Temple of Atal'Hakkar quest

## Changes Proposed:
-  Fixed the smartAI of Fel'zerul so he will cast break stuff when the quest Temple of Atal'Hakkar is completed

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/7884
- Closes https://github.com/chromiecraft/chromiecraft/issues/1722

## SOURCE:

https://www.wowhead.com/quest=1445/deprecated-the-temple-of-atalhakkar#comments:id=424027

## Tests Performed:

- .go c id 1443 
- .quest add 1445
- .quest complete 1445 and check the spell


## How to Test the Changes:


1.  .go c id 1443 
2.  .quest add 1445
3. .quest complete 1445 and check the spell

Video test

https://streamable.com/i5atw3



## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
